src/graph_engine.jl

```diff
@@ -949,6 +949,24 @@ Returns a collection of aliases for `fform` depending on the `backend`.
 aliases(backend, fform) = error("Backend $backend must implement a method for `aliases` for `$(fform)`.")
 aliases(model::Model, fform::F) where {F} = aliases(getbackend(model), fform)
 
+function add_vertex!(model::Model, label, data)
+    code = nv(model) + 1
+    model.graph.vertex_labels[code] = label
+    model.graph.vertex_properties[label] = (code, data)
+    Graphs.add_vertex!(model.graph.graph)
```

Comment:
This is an unsafe procedure to add a vertex to the model, the procedure from `MetaGraphsNext` does some additional checks that we don't have to do. This shaves a couple % off runtime

Reply:
Do you think its worth sacrificing checks and introduce an unsafe operation just to save a bit of run-time?  For a callee nothing suggest that this `add_vertex!(model, label, data)` is somehow unsafe? I don't mind if you are sure its worth it, but than maybe the comment should elaborate a bit what kind of checks have been removed in case if it causes problems in the future.

Reply:
I think so, `add_vertex!` for us should be internal, the checks that are passed are, for example, check if we don't exceed the maximum amount of vertices in the graph, or for edges, if the interfaces are in the right order (which we can assure we always pass in the right order).

Reply:
I mean, we can always call setindex! if we want a safe operation, it's just 25% slower across the entire model construction if we do this safe operation